test/unit/BondingManager.js

```diff
@@ -1799,26 +1800,45 @@ contract("BondingManager", accounts => {
             })
         })
 
-        describe("transcoder is not active", () => {
-            it("still decreases transcoder's bondedAmount", () => {
-                it("still decreases transcoder's bondedAmount", async () => {
-                    const startBondedAmount = (await bondingManager.getDelegator(transcoder))[0]
-                    await fixture.ticketBroker.execute(
-                        bondingManager.address,
-                        functionEncodedABI(
-                            "slashTranscoder(address,address,uint256,uint256)",
-                            ["address", "uint256", "uint256", "uint256"],
-                            [transcoder, constants.NULL_ADDRESS, PERC_DIVISOR / 2, 0]
-                        )
+        describe("transcoder is not active but registered", () => {
+            beforeEach(async () => {
+                await bondingManager.bond(2000, transcoder1, {from: transcoder1})
+                await bondingManager.transcoder(5, 10, {from: transcoder1})
+            })
+
+            it("still decreases transcoder's bondedAmount", async () => {
+                const startBondedAmount = (await bondingManager.getDelegator(transcoder1))[0]
+                await fixture.ticketBroker.execute(
+                    bondingManager.address,
+                    functionEncodedABI(
+                        "slashTranscoder(address,address,uint256,uint256)",
+                        ["address", "uint256", "uint256", "uint256"],
+                        [transcoder1, constants.NULL_ADDRESS, PERC_DIVISOR / 2, 0]
                     )
-                    const endBondedAmount = (await bondingManager.getDelegator(transcoder))[0]
+                )
+                const endBondedAmount = (await bondingManager.getDelegator(transcoder1))[0]
 
-                    assert.equal(endBondedAmount, startBondedAmount.div(2).toNumber(), "should decrease transcoder's bondedAmount by slashAmount")
-                })
+                assert.equal(endBondedAmount, startBondedAmount.div(new BN(2)).toNumber(), "should decrease transcoder's bondedAmount by slashAmount")
+            })
+
+            it("changes the total for the next round if transcoder will become active then", async () => {
```

Comment:
I think a clearer beginning of this description would be "decreases the total active stake for the next round..."

Reply:
fixed in e6d92cf